sdk/tests/system/cassettes/test_generic_agent/TestGenericAgent.test_continued_conversation.yaml

```diff
@@ -146,11 +151,10 @@ interactions:
         Y2hpbmUgd2hpY2ggZm9sbG93cyBpbnN0cnVjdGlvbnMgYW5kIHJldHVybnMgYSBzdW1tYXJ5IG9m
         IHlvdXIgYWN0aW9ucy4ifSwgeyJyb2xlIjogInVzZXIiLCAiY29udGVudCI6IFt7InR5cGUiOiAi
         dGV4dCIsICJ0ZXh0IjogIkhpISBteSBuYW1lIGlzIEx1a2UifV19LCB7InJvbGUiOiAiYXNzaXN0
-        YW50IiwgImNvbnRlbnQiOiAiSGVsbG8gTHVrZSEgSG93IGNhbiBJIGFzc2lzdCB5b3UgdG9kYXk/
-        In0sIHsicm9sZSI6ICJ1c2VyIiwgImNvbnRlbnQiOiBbeyJ0eXBlIjogInRleHQiLCAidGV4dCI6
-        ICJDYW4geW91IHNpbmcgbWUgSGFwcHkgQmlydGhkYXk/In1dfV0sICJtb2RlbCI6ICJncHQtNC12
-        aXNpb24tcHJldmlldyIsICJtYXhfdG9rZW5zIjogNDA5NiwgInN0cmVhbSI6IHRydWUsICJ0ZW1w
-        ZXJhdHVyZSI6IDAuM30=
+        YW50IiwgImNvbnRlbnQiOiAiIn0sIHsicm9sZSI6ICJ1c2VyIiwgImNvbnRlbnQiOiBbeyJ0eXBl
+        IjogInRleHQiLCAidGV4dCI6ICJDYW4geW91IHNpbmcgbWUgSGFwcHkgQmlydGhkYXk/In1dfV0s
+        ICJtb2RlbCI6ICJncHQtNC12aXNpb24tcHJldmlldyIsICJtYXhfdG9rZW5zIjogNDA5NiwgInN0
+        cmVhbSI6IHRydWUsICJ0ZW1wZXJhdHVyZSI6IDAuM30=
```

Comment:
ya know, since we are already customizing this, we should consider serializing the body into something readable.